app/models/object_event.rb

```diff
@@ -99,13 +99,13 @@ def event_types(types)
   private
   #
   # Generates the JSON representing the [ObjectEvent]
-  # @return [String] the JSON representing the ObjectEvent
+  # @return [Object] the JSON representing the ObjectEvent
   def to_object
-    ApiNew::ObjectEventsController.render 'api_new/object_events/generate', 
+    JSON.parse((ApiNew::ObjectEventsController.render 'api_new/object_events/generate', 
```

Comment:
jsonb Attributes now are de/serialized as objects, not string so we need to provider an object now.